.zenodo.json

```diff
@@ -43,9 +43,7 @@
       "affiliation": "CASUS, Helmholtz-Zentrum Dresden-Rossendorf",
       "name": "Stephan, Jan",
       "orcid": "0000-0001-7839-4386"
-    }
-  ],
-  "contributors": [
+    },
     {
       "name": "Gehrke, Valentin",
       "affiliation": "TU Dresden"
```

Comment:
Zenodo is invenio like rodare. It also supports contributors, but you forgot the `type` key:
https://github.com/nipreps/smriprep/blob/master/.zenodo.json#L36

Reply:
After how many features and PRs does one become author again? ;-)
Is 50 enough? :D https://github.com/alpaka-group/alpaka/pulls?q=is%3Apr+is%3Aclosed+author%3Aax3l

Reply:
Thanks for pointing to the mistake, I was searching for a schema but misinterpreted the zenodo error.

> After how many reviews, features and PRs does one become code co-author? ;-)

I decided to handle it that way and nobody was against it: https://github.com/alpaka-group/alpaka/pull/1236#issue-548172636
Everyone who contributed to the current release is `creators`, all other former contributors are `contributor`.

Reply:
Yep, I remembered it because this happened to me in the past, too.

Link: Ah I remember, looks legit. Thanks!